ddtrace/__init__.py

```diff
@@ -13,6 +13,11 @@
 # a global tracer instance with integration settings
 tracer = Tracer()
 
+
+class DDTraceDeprecationWarning(DeprecationWarning):
```

Comment:
you are probably going to get hit with circular imports here, might need to move to `ddtrace.internal.*` somewhere and expose publicly here

Reply:
Hm, but if we do that wouldn't the class path to the category change?

Reply:
We can move it to `ddtrace.internal.utils.deprecation.py`. We had something similar there before